README.md

```diff
@@ -41,7 +41,7 @@ print(ot.get_association('ENSG00000157764-EFO_0005803')[0])
 
 e_for_target = ot.get_evidence_for_target('BRAF')
 for e in e_for_target:
-    print(result_to_json(e))
+    print(json.dumps(e))
```

Comment:
json.dumps will not work for every response, since it might be a namedtuple. for the moment is better to keep using result_to_json until the code is promoted to stable.